### DISCUSSION TITLE
Duration issue - audio from cut mkv

### DISCUSSION THREAD

**Discussion by ghender3** [mifi/lossless-cut#848] [Q&A]
First - I think the software is great.

But I did have an issue recently:

I was working with an mkv video - roughly 2 hours long, and cut out a couple of (3-4 minutes) segments which contained music. My hope was to demux the new short mkv to isolate the audio, and then use an audio editor and/or convertor (Audacity, foobar 2K, dbPoweramp) to further process the audio.

However, whenever I attempted the above, each of the isolated audio pieces (which were FLAC) were shown by mediainfo (and any audio editor I tried) to have a _duration of ~2 hours. I_ tried isolating the audio using multiple approaches, but always got a "2 hour" file, which no audio editor/ convertor could handle.

I finally did what I wanted to do by returning to the original 2 hour video, avoiding doing anything to it with LosslessCut, demuxing to audio, and then editing the 2 hour audio file. 

Thanks for looking

**Comment by mifi** [comment]
Can you try disabling all tracks except 1 video and 1 audio track when cutting and see if that helps? Also if you could post a report from the help menu about your file it would be helpful

**Comment by ghender3** [comment]
OK, I think I still have the whole 2 hour video. Will attempt what you recommend.

Thanks for response

**Reply by ghender3** [reply]
OK, I have tried limiting the video to one video track and the one audio track I want to work with. I still have the same issue - I can create a 3 - 4 minute excerpt - a much smaller mkv, and it looks and sounds excellent. And MediaInfo reports that it is 3 minutes, 50 seconds, or whatever. Any tool that extracts tracks from an mkv gives me a Flac file with a reported duration of 2 hours, 15 minutes. Converting using mkvtoolnix gui to mka by using only the audio yields an mka that is fine and reported as 3 minutes, 50 seconds. But everything I have tried that can extract Flac from mka yields a Flac file reported as 2 hours, 15 minutes - the length of the file prior to Lossless-Cut. This includes opening with Lossless-Cut and exporting to raw FLAC. I am attaching info from the uncut original file obtained from the help menu.
[File.txt](https://github.com/mifi/lossless-cut/files/7230778/File.txt)

**Comment by ghender3** [comment]
I have obtained the file information on the brief .mka file. Attached as File 2. Also exported to raw FLAC using lossless-cut, file information on FLAC attached as File 3

[File 2.txt](https://github.com/mifi/lossless-cut/files/7231862/File.2.txt)
[File 3.txt](https://github.com/mifi/lossless-cut/files/7231863/File.3.txt)

**Comment by slrslr** [comment]
This issue (length of the file when flac is used) seems to be similar to https://github.com/mifi/lossless-cut/discussions/1320
